packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx

```diff
@@ -302,28 +302,14 @@ const ConfirmPlan = ({
           </Typography>
         </div>
       </div>
-      {paymentMethod === "crypto" && <div className={classes.rowBox}>
-        <Typography
-          variant="body1"
-          component="p"
-          className={classes.warningText}
-        >
-          <InfoCircleIcon className={classes.icon} />
-          <Trans>
-            Once you proceed, your account is expected to make a payment within 60 minutes. If no payment is received
-            , your plan will not change.
-          </Trans>
-        </Typography>
-      </div>
-      }
-      {isSubscriptionError &&
+      {subscriptionErrorMessage &&
```

Comment:
This should stay, that's a merge conflict gone wrong I think
```suggestion
      {paymentMethod === "crypto" && <div className={classes.rowBox}>
        <Typography
          variant="body1"
          component="p"
          className={classes.warningText}
        >
          <InfoCircleIcon className={classes.icon} />
          <Trans>
            Once you proceed, your account is expected to make a payment within 60 minutes. If no payment is received
            , your plan will not change.
          </Trans>
        </Typography>
      </div>
      }
      {subscriptionErrorMessage &&
```